src/test/java/org/prebid/server/bidder/consumable/ConsumableBidderTest.java

```diff
@@ -287,31 +285,12 @@ public void makeBidsShouldSkipDecisionsWithNullClearPrice() throws JsonProcessin
         assertThat(result.getErrors()).isEmpty();
     }
 
-    @Test
-    public void makeBidsShouldSkipDecisionsWithAbsentImpIdAndAddError() throws JsonProcessingException {
-        // given
-        final Map<String, ConsumableDecision> decisionMap = new HashMap<>();
-        decisionMap.put("firstImp", ConsumableDecision.builder().pricing(ConsumablePricing.of(10.5)).build());
-        decisionMap.put("missing_Imp", ConsumableDecision.builder().pricing(ConsumablePricing.of(1.1)).build());
-
-        final HttpCall<ConsumableBidRequest> httpCall = givenHttpCall(() -> ConsumableBidResponse.of(decisionMap));
-
-        // when
-        final Result<List<BidderBid>> result = consumableBidder.makeBids(httpCall,
-                givenBidRequestWithTwoImpsAndTwoFormats());
-
-        // then
-        assertThat(result.getErrors()).hasSize(1)
-                .containsOnly(BidderError.badServerResponse("ignoring bid id=request_id, request doesn't contain any "
-                        + "impression with id=missing_Imp"));
-        assertThat(result.getValue()).hasSize(1);
-    }
-
     @Test
     public void makeBidsShouldReturnBannerBidWithExpectedFields() throws JsonProcessingException {
         // given
         final HttpCall<ConsumableBidRequest> httpCall = givenHttpCall(identity(),
                 decision -> decision.pricing(ConsumablePricing.of(11.1)).adId(123L)
+                        .width(120).height(90)
```

Comment:
You used same values, which we used in `Banner.format[0]` so for better understanding, lets use diffrent one.

(Also correct line 306)